assignments/prototypes.js

```diff
@@ -15,13 +15,29 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
+const GameObject (){
+  this.createdAt: "Date()",
```

Comment:
Awesome that you're using the Date constructor here! If you'd like for this to be an actual Date object, you can remove the quotes. Right now, createdAt will always be a string with the value of "Date()"